x-pack/plugins/observability_solution/investigate/common/types.ts

```diff
@@ -36,13 +35,12 @@ export interface InvestigateWidget<
   TData extends Record<string, any> = {}
 > {
   id: string;
-  created: number;
-  last_updated: number;
+  createdAt: number;
+  createdBy: string;
```

Comment:
@kdelemme Maybe we want to use here `createdBy: user.username` similar to what you use [here](https://github.com/elastic/kibana/pull/190852/files#diff-6d6b56d1cb8b912a74f6ee48feaff0633296fa0b9085d7757ca08aa5bdfd491aR50)?